reviewers.yaml

```diff
@@ -20,8 +20,6 @@ ggreenway:
   opsgenie: Greg
   slack: U78MBV869
 jmarantz:
-  maintainer: true
-  opsgenie: Joshua
```

Comment:
I am not sure if this is correct; leaving myself in as a reviewer (e.g. for stats changes) but removing the maintainer bit and opsgenie rotation.

Reply:
this is used to send you slack notifications and associate your github username in slack - guessing you probably want to leave this

Reply:
Sorry, not following. Are you referring to the 'maintainer: true' line or the 'opsgenie: Joshua' line?

I was assuming that making me not be a 'maintainer' would mean I wouldn't get notified when maintaners are referenced in a github client.

I was assuming 'opsgenie' referred to me being on maintainers triage rotation every N weeks.

What are you suggesting I leave in?

Reply:
apologies - ignore me - i see you did leave it - you just removed the opsgenie - apologies again